tests/integration/cgroups_fetcher_mock_test.go

```diff
@@ -0,0 +1,38 @@
+package biz
```

Comment:
This biz package was already here, and there is no benefit from it, this biz package is living completely in the tests/integration folder and has no access to the biz package in the source, if you see, in the test the calls to that package are done biz.blabla... so this package can be called as we want because it will always be indepenedent.

Reply:
> this biz package is living completely in the tests/integration folder and has no relation nor access to the biz package in the source

I totally miss this , cool